setup/azurecmdfiles.wxi

```diff
@@ -4,6 +4,71 @@
         <DirectoryRef Id="PowerShellFolder">
             <Directory Id="dirB60B5B1F56FA5598EFFC960CA393FDCE" Name="ResourceManager">
```

Comment:
As discussed previously, you need to add your module to AzureRm.psd1 in tools\AzureRM\AzureRM.psd1